src/ZombieDriver.php

```diff
@@ -236,7 +236,16 @@ public function setRequestHeader($name, $value)
      */
     public function getResponseHeaders()
     {
-        return (array) $this->server->evalJS('browser.window._response.headers', 'json');
+        $js = <<<JS
+        if (true == zombieVersionCompare('4.0.0', '>=')) {
```

Comment:
> could we rely on some feature detection rather than version sniffing ?

Any example of that, that you can link to?

In this case the `headers` property always exists, but it's contents is different. Not sure what to check.